testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -542,33 +550,47 @@ impl RunLoop {
         let (coordinator_thread_handle, attachments_rx) =
             self.spawn_chains_coordinator(&burnchain_config, coordinator_receivers);
         self.instantiate_pox_watchdog();
+        self.start_prometheus();
 
         // We announce a new burn block so that the chains coordinator
         // can resume prior work and handle eventual unprocessed sortitions
         // stored during a previous session.
         coordinator_senders.announce_new_burn_block();
 
-        // Wait for some sortitions!
-        let mut burnchain_tip = burnchain
-            .wait_for_sortitions(None)
-            .expect("Unable to get burnchain tip");
+        // Make sure at least one sortition has happened
+        let sortdb = burnchain.sortdb_mut();
+        let (rc_aligned_height, sn) =
+            RunLoop::get_reward_cycle_sortition_db_height(&sortdb, &burnchain_config);
+
+        let burnchain_tip_snapshot = if sn.block_height == burnchain_config.first_block_height {
+            // need at least one sortition to happen.
+            burnchain
+                .wait_for_sortitions(Some(sn.block_height + 1))
+                .expect("Unable to get burnchain tip")
+                .block_snapshot
+        } else {
+            sn
+        };
 
         // Boot up the p2p network and relayer, and figure out how many sortitions we have so far
         // (it could be non-zero if the node is resuming from chainstate)
         let mut node = StacksNode::spawn(
             self,
-            Some(burnchain_tip.clone()),
+            Some(burnchain_tip_snapshot),
             coordinator_senders.clone(),
             attachments_rx,
         );
-        let sortdb = burnchain.sortdb_mut();
-        let mut sortition_db_height = RunLoop::get_sortition_db_height(&sortdb, &burnchain_config);
+
+        // Wait for all pending sortitions to process
+        let mut burnchain_tip = burnchain
+            .wait_for_sortitions(None)
+            .expect("Unable to get burnchain tip");
```

Comment:
Do you still think we should do this?

Reply:
Yes. This runs once all the rest of the threads are booted up and running, so the net effect here is that any unprocessed sortitions get processed before the node downloads more Bitcoin blocks.

Reply:
Okay, works for me